### DISCUSSION TITLE
New release request

### DISCUSSION THREAD

**Discussion by FrodeBjerkholt** [BerkleyTechnologyServices/restdocs-spec#135] [General]
Thanks for a great maven-plugin! -Will there be a new release soon? The reason for asking, is that a bug fix in the restdocs-api-spec 0.11.4 resolves some issues for me.  Best Regards, Frode Bjerkholt

**Comment by klieber** [comment]
@FrodeBjerkholt Sorry, I just noticed this now.  I will try to get a release out today.

**Reply by klieber** [reply]
0.21 should be available in maven central now
